examples/server.rs

```diff
@@ -15,7 +15,7 @@ macro_rules! try_return(
     ($e:expr) => {{
         match $e {
             Ok(v) => v,
-            Err(e) => { error!("Error: {}", e); return; }
+            Err(e) => { println!("Error: {}", e); return; }
```

Comment:
I did that on purpose, since it seemed better for a simple example to actually print errors.